src/Tests/Compile/CompileEnv.test.ts

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {strict as assert} from 'assert';
+
+import {CompilerBase} from '../../Backend/Compiler';
+import {Toolchains} from '../../Backend/Toolchain';
+import {ToolchainInfo} from '../../Backend/Toolchain';
+import {DebianToolchain} from '../../Backend/ToolchainImpl/DebianToolchain';
+import {CompileEnv} from '../../Compile/CompileEnv';
+import {Logger} from '../../Utils/Logger';
+
+class MockCompiler extends CompilerBase {
+  // TODO: What toolchain is necessary as tests?
+  installedToolchain: DebianToolchain;
+  availableToolchain: DebianToolchain;
```

Comment:
IMHO it's better to move this into local var inside constructor().

Reply:
Sorry, I don't understand what you mean. Could you give me a detail explanation?

Reply:
I modified my comment. Sorry for wrong English expression. 😹